roles/lib_openshift/src/class/oc_route.py

```diff
@@ -55,7 +55,13 @@ def create(self):
 
     def update(self):
         '''update the object'''
-        # need to update the tls information and the service name
+        # when the host attribute is being updated, we need to delete and recreate
```

Comment:
@detiber, I agree it shouldn't be.  https://bugzilla.redhat.com/show_bug.cgi?id=1426666

If you could test this and confirm.  I've tested on 3.4 and 3.5 and it is immutable.